boot/boot_serial/src/boot_serial.c

```diff
@@ -28,13 +28,13 @@
 #include "zcbor_encode.h"
 
 #ifdef __ZEPHYR__
-#include <sys/reboot.h>
-#include <sys/byteorder.h>
-#include <sys/__assert.h>
-#include <drivers/flash.h>
-#include <sys/crc.h>
-#include <sys/base64.h>
-#include <hal/hal_flash.h>
+#include <zephyr/sys/reboot.h>
+#include <zephyr/sys/byteorder.h>
+#include <zephyr/sys/__assert.h>
+#include <zephyr/drivers/flash.h>
+#include <zephyr/sys/crc.h>
+#include <zephyr/sys/base64.h>
+#include <zephyr/hal/hal_flash.h>
```

Comment:
hmm I can't see hal_flash.h in Zephyr.

Reply:
sorry my bad, it's mcuboot specific, the current copy in upstream zephyr has it out of the __ZEPHYR__ block, things have been shuffled around with the ESP32 port. I'll drop the prefix from this specific one.